src/Bindings/ManualBindings_World.cpp

```diff
@@ -890,14 +890,19 @@ static int tolua_cWorld_SpawnSplitExperienceOrbs(lua_State* tolua_S)
 	}
 
 	cWorld * self = nullptr;
-	Vector3d * Position;
+	Vector3d * Position = nullptr;
 	int Reward;
 	L.GetStackValues(1, self, Position, Reward);
 	if (self == nullptr)
 	{
 		tolua_error(tolua_S, "Invalid 'self' in function 'SpawnSplitExperienceOrbs'", nullptr);
 		return 0;
 	}
+	if (Position == nullptr)
+	{
+		tolua_error(tolua_S, "Error in function 'SpawnSplitExperienceOrbs' arg #2. Value must not be nil.", nullptr);
```

Comment:
The message might be misleading, `Position == nullptr` due to other causes as well - bad parameter type. A better message would be `Value must be a valid Vector3`, or even better `Expected Vector3, got XYZ instead`.

Reply:
Oh, I thought that the `CheckParamUserType(2, "Vector3<double>")` at the top of the function would check for that kind of type mismatch, and that only `nil` was slipping through because it's a valid `Vector3<double>`.

Reply:
You're right, I've missed that. Sorry.